sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -111,7 +126,13 @@ def __init__(
         **kwargs
     ) -> None:
         self._checkpoint_store = kwargs.pop("checkpoint_store", None)
-        self._load_balancing_interval = kwargs.pop("load_balancing_interval", 10)
+        self._load_balancing_interval = kwargs.pop("load_balancing_interval", None) or 10
+        self._partition_ownership_expiration_interval = kwargs.pop(
+            "partition_ownership_expiration_interval", None
+        ) or 6*self._load_balancing_interval
+        load_balancing_strategy = kwargs.pop("load_balancing_strategy", None) or LoadBalancingStrategy.GREEDY
+        self._load_balancing_strategy = LoadBalancingStrategy(load_balancing_strategy) if load_balancing_strategy \
```

Comment:
What if the user already passes in an instance of `LoadBalancingStrategy` - does this still work?

Reply:
Yes, my test shows that LoadBalancingStrategy(LoadBalancingStrategy.GREEDY) returns LoadBalancingStrategy.GREEDY